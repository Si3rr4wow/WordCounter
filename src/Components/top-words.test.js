import React from 'react'
import renderer from 'react-test-renderer'
import TopWords from './top-words'

it('Renders the top words display', () => {
  const tree = renderer
    .create(<TopWords topWords={{ a: 3, b: 2, c: 1 }}/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
