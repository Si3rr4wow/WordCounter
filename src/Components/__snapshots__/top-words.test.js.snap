// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the top words display 1`] = `
Array [
  <h3>
    Top Words:
  </h3>,
  <p>
    a
    : 
    3
  </p>,
  <p>
    b
    : 
    2
  </p>,
  <p>
    c
    : 
    1
  </p>,
]
`;
